src/main/java/Main.java

```diff
@@ -0,0 +1,96 @@
+import java.io.IOException;
+import java.util.Scanner;
+
+public class Main {
+    public static void main(String[] args) throws DukeException{
+
```

Comment:
Good OOP Design Joshua! I also like the way you conform to Java Coding Standards in your naming of methods and variables. Keep it up!